lib/middleware/check-progress.js

```diff
@@ -18,7 +18,7 @@ module.exports = function checkProgress(route, controller, steps, start) {
         var potentialPaths = _.pluck(controller.options.forks, 'target')
             .concat(controller.options.next);
 
-        var nextStep = controller.getNextStep(req, res).replace(/\/edit$/, '');
+        var nextStep = controller._getForkTarget(req, res);
```

Comment:
I can't think of anything else other than your suggestion @joefitter (`getForkTarget` calling `_getForkTarget`). This way we follow the common naming convention for public and private methods. Although the methods could easily be overridden I'm not sure why they would need to.